presto-release/src/main/java/com/facebook/presto/release/ForPresto.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.release;
+
+import javax.inject.Qualifier;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Retention(RUNTIME)
+@Target({FIELD, PARAMETER, METHOD})
+@Qualifier
+public @interface ForPresto
```

Comment:
It means "something related to Presto". Admitted, it is vague without any context. In this PR, there are 2 usages:
- `@ForPresto Git git`: a object to perform git operations on presto repo
- `@ForPresto LocalRepository repository`: represents the local git repo for Presto

Comparatively, we will also have `@ForPrestoFacebook` in a to-be-exist presto-facebook-release module.

We have 60+ of those annotations in presto, but none has comment - I think their exact meanings are better documented by usages in code rather than comment?

Reply:
I'll leave it up to you as to whether you want to add comment.

When doing code review I have an amount of effort I put into understanding what something does. Once I exceed that level of effort I start to think about whether a comment will save enough time for the people that come later. Code is write once read many so IMO the ROI is there.

There are some dependency injection related idioms here that still not very familiar with so it tripped me up enough to ask.